http/client.go

```diff
@@ -451,7 +451,7 @@ func (c *Client) Headstore() ds.Read {
 	panic("client side database")
 }
 
-func (c *Client) Events() events.Events {
+func (c *Client) Events() *event.Bus {
 	panic("client side database")
```

Comment:
thought: This can/should probably be implemented for http/cli at some point, half tempted to ask you to link this panic to a github issue.

Reply:
I'm planning on adding this functionality in a follow up PR. One of my design goals was to allow the CLI to subscribe to and print the events as they happen.

Reply:
Sweet, sounds good :)